src/main/java/com/ericsson/eiffelcommons/helpers/JenkinsXmlData.java

```diff
@@ -37,6 +40,15 @@
     private static final String XML_VERSION = "<?xml version='1.1' encoding='UTF-8'?>";
     private static final String HUDSON_PARAMETERS_DEFINITION_KEY = "hudson.model.ParametersDefinitionProperty";
     private static final String PARAMETER_DEFINITION_KEY = "parameterDefinitions";
+    private static final String GROOVY_SCRIPT_SCRIPT_PLUGIN_VERSION = "2.1";
+    private static final String GROOVY_SCRIPT_SECURITY_PLUGIN_VERSION = "1.51";
+
+    private String hudsonPluginsGroovyGroovyKey = "hudson.plugins.groovy.Groovy plugin='groovy@%s'";
```

Comment:
Why groovy two times in the name?

Reply:
it reflects the xml node name, it has groovy 2 times.